### PR TITLE
tools: modified rtk_player for gear switch scenarios

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/tools/record_play/rtk_player.py [hunk 1]
```diff
@@ -168,6 +168,25 @@ def closest_time(self):
 
         return closest_time
 
+    def next_gear_switch_time(self, start, end):
+        for i in range(start, end):
+            # trajectory with gear switch
+            # include gear_neutral at the beginning of a trajectory
+            if((self.data['gear'][i] == 1 or self.data['gear'][i] == 2)
+                    and (self.data['gear'][i + 1] != self.data['gear'][i]) ):
+                print("enter i in while loop: [", i, "]")
```
Comment (JasonXu12) [inline modules/tools/record_play/rtk_player.py]: please use self.logger.info/debug to show information instead of print

Comment (sjiang2018) [reply modules/tools/record_play/rtk_player.py]: fixed. Thank you.

File: modules/tools/record_play/rtk_player.py [hunk 2]
```diff
@@ -135,22 +135,32 @@ def restart(self):
                          (self.start, self.closestpoint))
 
         self.closestpoint = self.closest_dist()
+        self.logger.debug("replan!")
         self.start = max(self.closestpoint - 1, 0)
+        self.logger.debug("replan_start:", self.start)
         self.starttime = cyber_time.Time.now().to_sec()
-        self.end = min(self.start + 1000, len(self.data) - 1)
+        self.logger.debug("at time", self.starttime)
+        # self.end = min(self.start + 1000, len(self.data) - 1)
```
Comment (Capri2014) [inline modules/tools/record_play/rtk_player.py]: @siyangy remove this commented code please

Comment (Capri2014) [reply modules/tools/record_play/rtk_player.py]: @sjiang2018 remove this commented code please

**Pull request merged**